### PR TITLE
Make view survey button show up for CSA

### PR DESCRIPTION
**Pull request by dmcavoy**
Natalia [reported](https://codedotorg.slack.com/archives/CU8HMEE06/p1654108552905129) some parts of the workshop dashboard survey views not showing for CSA workshops. This PR fixes those missing things. 

Show View Survey Button and Pre Survey Tab when viewing workshop:

Screen Shot 2022-06-01 at 9 48 02 PM

Roll Up Tabs Showing for CSA

Screen Shot 2022-06-01 at 10 10 07 PM

## Links

- https://codedotorg.atlassian.net/browse/PLAT-1866

## Testing story

- Local testing

### Follow Up

- This area of the code base could use some organizing so that we are using our constants more consistently so that when we add new courses it is as easy as adding it in the shared_workshop_constants.rb to get it to have the right settings

### CODE REVIEW

File: apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx [hunk 1]
```diff
@@ -185,9 +188,7 @@ export default class EnrollmentsPanel extends React.Component {
   };
 
   getViewSurveyUrl = (workshopId, course, subject, lastSessionDate) => {
-    if (
-      !['CS Discoveries', 'CS Principles', 'CS Fundamentals'].includes(course)
-    ) {
+    if (!ActiveCoursesWithSurveys.includes(course)) {
```
Comment (megcrenshaw) [inline apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx:191]: It looks like the course itself comes in as the long string `CS Discoveries`. Is that getting mapped with `COURSE_CSD` via the `ACTIVE_COURSES` array? Not sure how that works

Comment (dmcavoy) [reply apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx:191]: Things like `COURSE_CSD` are set up here: https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/shared_constants/pd/shared_workshop_constants.rb#L4-L9. I then reused them in `ACTIVE_COURSES_WITH_SURVEYS` and then we generate the matching javascript constant `ActiveCoursesWithSurveys`. There is definitely more clean up that can be done around these constants but I think thats something that shouldn't block the fix on this

File: lib/cdo/shared_constants/pd/shared_workshop_constants.rb [hunk 2]
```diff
@@ -228,11 +228,18 @@ module SharedWorkshopConstants
     }.freeze
 
     ACTIVE_COURSE_WORKSHOPS = {
-      CSD: 'CS Discoveries',
-      CSP: 'CS Principles',
-      CSA: 'Computer Science A'
+      CSD: COURSE_CSD,
```
Comment (dmcavoy) [inline lib/cdo/shared_constants/pd/shared_workshop_constants.rb:231]: This was just a clean up. There are a lot of constants in this area of the code that could use to be cleaned up

**Review by megcrenshaw — APPROVED** [review]
Looks great, Dani!

**Pull request merged**